### ISSUE TITLE
Object detection project (root architecture) using Tensorflow + Keras. Image sample size for accurate training of model?

### ISSUE THREAD

**Issue by darvida** [matterport/Mask_RCNN#986]
Im currenty working on a project at University, where we are using python + tensorflow and keras to train an image object detector, to detect different kinds of roots that the plant arabidopsis have. Our current ressults are pretty bad but we do only have about 100 images to train the model with at this moment. We are currently working on cultuvating more plants in order to get more images(more data) to train the model.tensorflow.

We are looking to detect three object clases: stem, main root and secondary root. But the model detects main roots incorrectly where the secondary roots located. It whould be able to detect something like this: [https://imgur.com/a/rQ1YHH3](Root detection example)

What is the usual sample size that is used to train a neural network accurate results?

**Comment by maxfrei750** [comment]
Could you please also upload an example of the masks that you are using for the training? Ideally the masks associated with the image that you already uploaded.

**Comment by darvida** [comment]
Hey i have uploaded a .rar file with the training images and the masks in .json files
Images + json file:[http://www.filedropper.com/train2](url)

**Comment by theronic** [comment]
@darvida, I can't access your masks link. Can you reupload, maybe to Github?
